includes/blocks/course-theme/class-course-theme.php

```diff
@@ -56,9 +58,14 @@ protected function should_initialize_blocks() {
 	 * Initializes the blocks.
 	 */
 	public function initialize_blocks() {
-		$prev = new Prev_Lesson();
-		$next = new Next_Lesson();
-		new Prev_Next_Lesson( $prev, $next );
-		new Quiz_Button();
+		if ( 'lesson' === get_post_type() ) {
+			new Prev_Lesson();
+			new Next_Lesson();
+			new Prev_Next_Lesson();
+			new Course_Progress_Counter();
+			new Quiz_Button();
+		} elseif ( 'quiz' === get_post_type() ) {
```

Comment:
Minor thing but maybe you could use a `$post_type` variable so we don't call `get_post_type()` twice?

Reply:
Just noting that this doesn't really have a performance implication (most things in Wordpress are cached for subsequent access, and this one is just property access on a global object anyway), so if it keeps the code cleaner there isn't much need for a variable.

Reply:
Good to know, thanks!